### PR TITLE
KOGITO-3909: Standalone DMN editor missing isDirty indication on datatype or included models change

### PR DESCRIPTION
**Pull request by danielzhe**
See: https://issues.redhat.com/browse/KOGITO-3909

VSIX: https://drive.google.com/file/d/1zqqMnJVDOj9mlzPALHPMHlpvwFPiAi0N/view?usp=sharing

VSIX v 2.0: https://drive.google.com/file/d/1q_b7CcAkNvA6YMbzbHlre2cxEwBF-xn2/view?usp=sharing

Known issues found during this PR, but not related to it:
https://issues.redhat.com/browse/KOGITO-5737
https://issues.redhat.com/browse/KOGITO-5727

### CODE REVIEW

**Review by karreiro — COMMENTED** [review]
Thank you, @danielzhe! Great stuff :-)

I've noticed that when I rename an included model, it seems like the it doesn't undo. Is this something we could handle?

dssdds

Thanks again for the great work!

**Comment by danielzhe** [comment]
Yes, I gonna check this and the failing tests. Thank you, @karreiro

File: kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/included/commands/RenameIncludedModelCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    public void testExecuteWhenIncludedModelIsValid() {
+        doTestExecute("newName", "newName");
+    }
+
+    @Test
+    public void testExecutedWhenIncludedModelIsValidWithWhitespace() {
+        doTestExecute("   newName   ", "newName");
+    }
+
+    public void doTestExecute(final String newName, final String expectedName) {
```
Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/included/commands/RenameIncludedModelCommandTest.java:81]: could we please make this method private?

**Review by jomarko — COMMENTED** [review]
@danielzhe thank you for PR, one comment to code, now I will proceed with a manual review.

**Comment by jomarko** [comment]
@karreiro @danielzhe 

as we mention Undo for **Included Models** here https://github.com/kiegroup/kogito-editors-java/pull/99#pullrequestreview-735174712 I checked it also for **Data types** tab, however there I am not able to **Undo** or **Redo** a command. Is that expected? Asking as we introduced quite a lot of commands, what I thought (from my previous reviews) should make undo redo possible, thank to our framework.

**Review by jomarko — APPROVED** [review]
Thank you all

**Comment by jomarko** [comment]
@karreiro @danielzhe can we merge this PR?

**Comment by karreiro** [comment]
Thanks, @jomarko.

**Pull request merged**